### PR TITLE
v2

### PR DESCRIPTION
**Pull request by LeoQQLL**
Hello

### CODE REVIEW

**Comment by LeoQQLL** [comment]
I add the package

**Comment by LeoQQLL** [comment]
Please check

File: test.py [hunk 1]
```diff
@@ -1 +1,2 @@
-import numpy
\ No newline at end of file
+import numpy
+import scipy
```
Comment (LeoQQLL) [inline test.py:2]: add a new package

Comment (LieQU) [inline test.py:2]: OK

**Review by LieQU — COMMENTED** [review]
OKOK

**Review by LeoQQLL — COMMENTED** [review]
pending

**Review by LieQU — APPROVED** [review]

**Pull request merged**